easybuild/easyconfigs/d/double-down/double-down-1.1.0-foss-2022a.eb

```
@@ -0,0 +1,37 @@
+name = 'double-down'
+version = '1.1.0'
+moduleclass = 'phys'
+
+homepage = 'https://github.com/pshriwise/double-down'
+description = "double-down is a double precision interface to Embree via the Mesh Oriented dAtaBase (MOAB)."
+software_license = 'LicenseGCC'
+software_license_urls = ['https://github.com/pshriwise/double-down/blob/develop/LICENSE']
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [{
+    'filename': 'double-down.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/pshriwise',
+        'repo_name': 'double-down',
+        'tag': f'v{version}'
+    },
+}]
```

With `git_config` you can't add checksums. So better download the file from the releases:
```suggestion
source_urls = ['https://github.com/pshriwise/double-down/archive/']
sources = ['v%(version)s.tar.gz']
```

---

Makes sense. I've done this for the other easyconfigs that don't require submodules. The ones that do still use `git_config`.

---

@Flamefire, shouldn't checksums still work if you're pulling from a specific tag?

---

> I've done this for the other easyconfigs that don't require submodules. The ones that do still use git_config.

You can still use the tarballs by providing the sources and the directory to extract to. Often you don't even need those as many vendor dependencies as submodules and we can just provide them ourselves.

For OpenMC I see (ticks show available ECs):
- [x] Catch2
- [x] fmt
- [ ] gsl-lite
- [x] pugixml
- [x] xtensor
- [ ] xtl

So for most of them you don't need the submodules. For the remaining 2 it might make sense to create ECs or (for now at least) download the submodules

> shouldn't checksums still work if you're pulling from a specific tag?

No because a plain clone (done by EB with `git_config`) isn't fully deterministic.